tests/functional/test_project.py

```diff
@@ -46,6 +46,7 @@ def test_extract_manifest(dependency_config, project_manager):
     # NOTE: Only setting dependency_config to ensure existence of project.
     manifest = project_manager.extract_manifest()
     assert type(manifest) == PackageManifest
+    assert type(manifest.compilers) == list
```

Comment:
can we add an assert, like make sure the list has a single item in it with an ape-pm noticeable property

Reply:
perfect thank you 👍

Reply:
Not sure this is a really good test